diff -urN evms-0.9.2.org/engine/make.rules.in evms-0.9.2/engine/make.rules.in
--- evms-0.9.2.org/engine/make.rules.in	Tue Mar 19 13:55:03 2002
+++ evms-0.9.2/engine/make.rules.in	Tue Mar 19 14:07:18 2002
@@ -4,11 +4,11 @@
 subdirs = @subdirs@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
-bindir = $(exec_prefix)/bin
-sbindir = ${exec_prefix}/sbin
-infodir = $(prefix)/info
-libdir = $(prefix)/lib
-mandir = $(prefix)/man
+bindir = @bindir@
+sbindir = @sbindir@
+infodir = @infodir@
+libdir = @libdir@
+mandir = @mandir@
 date := $(shell date +%D)
 
 HOMEDIR=@homedir@
@@ -24,7 +24,7 @@
 CPP		= @CPP@
 CC	  	= @CC@
 LIBDIRS		= -L$(HOMEDIR)/Engine -L$(HOMEDIR)/dlist
-INCLUDES	= -I$(HOMEDIR)/include -I$(HOMEDIR)/dlist -I$(KERNELDIR)/include -I/usr/include -I/usr/local/include
+INCLUDES	= -I$(HOMEDIR)/include -I$(HOMEDIR)/dlist -I$(KERNELDIR)/include -I/usr/include/ncurses
 DEFINES		= @evmsdefs@ -DMAJOR_VERSION=$(MAJOR_VERSION) -DMINOR_VERSION=$(MINOR_VERSION) -DPATCH_LEVEL=$(PATCH_LEVEL) -DVERSION=\"$(VERSION)\" -DDATE=\"$(date)\"
 
 GTK_LIBS	= @gtklibs@
@@ -32,7 +32,7 @@
 ENGINE_DEBUG	= @debug@
 EFENCE		= @efence@
 
-EVMSCFLAGS	= -Wall -fPIC -O2 $(ENGINE_DEBUG) $(INCLUDES) $(DEFINES)
+EVMSCFLAGS	= -Wall -fPIC $(OPT) $(ENGINE_DEBUG) $(INCLUDES) $(DEFINES)
 SO_CFLAGS       = -shared $(EFENCE) -lc -ldl -L$(HOMEDIR)/dlist -ldlist
 
 .c.o:
